samples/trigger-customizations/600-event-sender.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: event-sender
+  namespace: trigger-demo
+spec:
+  restartPolicy: Never
+  containers:
+    - name: event-sender
+      image: ko://knative.dev/eventing-rabbitmq/samples/trigger-customizations/event-sender
```

Comment:
Same comment here with the [ping source](https://github.com/knative/eventing/blob/main/config/core/resources/pingsource.yaml) or the [Container Source](https://knative.dev/docs/eventing/custom-event-source/containersource/)

Reply:
For the sender, I looked at the test-images provided by eventing-core but images like the heartbeat required a period > 0. Ping source relies on a cron expression as well. So they send a message per x seconds. I needed something that would spam x messages as quickly as possible.

Reply:
With the container source you can set ms too (just In case) but I'm approving this :)

Reply:
If we can avoid maintaining another test_image I'm all for it. I saw the container source using the heartbeats test_image as well n was deterred by [this](https://github.com/knative/eventing/blob/main/test/test_images/heartbeats/main.go#L107-L112). Maybe I can modify that to take a string for duration instead like `5s` or `5n` which will help us reuse it

Reply:
Was in the middle of making the switch then I realized with heartbeats, you can't limit the number of messages. Ideally I want something like 10 messages or so. With creating heartbeat container and setting a low period like 1ms, it becomes hard to observe events at the display with all of the messages coming in very quickly without and end.

Reply:
In this case, I'd rather to explain something like logs -f to the user than to have another image to maintain. But I'll leave it up to you to decide. Either way, I'm approving :)